patches/server/1015-Add-mob-spawner-ranges-changes-and-API.patch

```diff
@@ -0,0 +1,119 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: DavidTs93 <david.ts93@gmail.com>
+Date: Fri, 7 Apr 2023 19:28:43 +0300
+Subject: [PATCH] Add mob spawner ranges changes and API
+
+
+diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+index 019d3bbd78fb0b06861979d223915fedb6c99442..daf1747955e175d1e37d631fcbd574575a667433 100644
+--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+@@ -307,6 +307,18 @@ public class GlobalConfiguration extends ConfigurationPart {
+                 executor.setMaximumPoolSize(_chatExecutorMaxSize);
+             }
+         }
++
++        public DefaultMobSpawnerValues defaultMobSpawnerValues;
++
++        public class DefaultMobSpawnerValues extends ConfigurationPart {
++            public Range range;
++            
++            public class Range extends ConfigurationPart {
++                public int horizontal = 4;
++                public int vertical = 1;
++            }
++        }
++
```

Comment:
This should be in the config patch ([0005-Paper-config-files.patch](https://github.com/PaperMC/Paper/blob/master/patches/server/0005-Paper-config-files.patch))